shared-services/src/files/cloudinary-file.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from "@nestjs/common"
```

Comment:
Why do we have dependencies on nestjs and backend/core?

Reply:
I removed the dependency on the backend since we aren't using the SharedModule for environment variables. For the Module import from NestJs, this new module is not used yet so I can remove the entire module if you prefer and only add it in the future if necessary.

Reply:
I think it makes sense to omit for now and revisit if it's needed later

Reply:
Done